src/test/end-to-end/helpers/etCaseHepler.js

```diff
@@ -14,8 +14,8 @@ const { I } = inject()
 const location = 'ET_EnglandWales';
 const etDataLocation = dataLocation.data;
 const s2sBaseUrl = `http://rpe-service-auth-provider-${env}.service.core-compute-${env}.internal/lease`;
-const token = totp(testConfig.TestCcdGwSecret, { digits: 6, period: 30 });
-const oneTimepwd = token;
+//const token = totp(testConfig.TestCcdGwSecret, { digits: 6, period: 30 });
```

Comment:
Is this no longer needed?

Reply:
This has been moved into the method as the OTP has to be recreated every time that the request has to be made...Please refer to Line 46...